app/styles/profile.css

```diff
@@ -101,6 +102,20 @@
 
 .edit-btn {
   padding: 0;
+  margin-top: 0.5rem;
+  color: var(--profile-edit-btn-clr);
+  font-weight: 600;
+  font-size: 1rem;
+}
+
+.profile-edit-button {
+  padding-inline: 5px;
+  padding-block: 3px;
```

Comment:
- these seem random to me can we use multiples of 4 (4, 8, 12...) as we follow it in our design system?

Reply:
Hi @yesyash I've fixed this and padding is now in multiple of 4.